core/src/main/java/com/google/instrumentation/trace/SpanId.java

```diff
@@ -62,22 +87,41 @@ public static SpanId fromBytes(byte[] bytes) {
    * @return a valid new {@code SpanId}.
    */
   public static SpanId generateRandomId(Random random) {
-    byte[] bytes = new byte[SPAN_ID_SIZE];
+    byte[] bytes = new byte[SIZE];
     do {
       random.nextBytes(bytes);
     } while (Arrays.equals(bytes, INVALID.bytes));
     return new SpanId(bytes);
   }
 
   /**
-   * Returns the 8-bytes array representation of the {@code SpanId}.
+   * Returns the byte representation of the {@code SpanId}.
    *
-   * @return the 8-bytes array representation of the {@code SpanId}.
+   * @return the byte representation of the {@code SpanId}.
    */
   public byte[] getBytes() {
-    return Arrays.copyOf(bytes, SPAN_ID_SIZE);
+    return Arrays.copyOf(bytes, SIZE);
   }
 
+  /**
+   * Copies the byte array representations of the {@code SpanId} into the {@code dest} beginning at
+   * the {@code destPos} position.
```

Comment:
Use of Pos (position) terminology, where elsewhere Offset is used. Make them consistent, I think offset is better.